editor/components/inserter/group.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import { isEqual } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { NavigableMenu } from '@wordpress/components';
+import { BlockIcon } from '@wordpress/blocks';
+
+function deriveActiveBlocks( blocks ) {
+	return blocks.filter( ( block ) => ! block.disabled );
+}
+
+export default class InserterGroup extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.onNavigate = this.onNavigate.bind( this );
+
+		this.activeBlocks = deriveActiveBlocks( this.props.blockTypes );
+		this.state = {
+			current: this.activeBlocks.length > 0 ? this.activeBlocks[ 0 ].name : null,
+		};
+	}
+
+	componentWillReceiveProps( nextProps ) {
+		if ( ! isEqual( this.props.blockTypes, nextProps.blockTypes ) ) {
+			this.activeBlocks = deriveActiveBlocks( nextProps.blockTypes );
+			// Try and preserve any still valid selected state.
+			const current = this.activeBlocks.find( block => block.name === this.state.current );
```

Comment:
`Array#find` is not polyfilled and is not supported by IE11. This results in crashes in IE11.

Reply:
Should we provide a runtime warning in dev env?

Reply:
Per https://github.com/WordPress/gutenberg/pull/746#issuecomment-300529051, I think eventually we'll just want to pull in the polyfill, ideally once Babel 7.x lands.